model/src/main/kotlin/CuratedPackage.kt

```diff
@@ -54,7 +54,12 @@ data class CuratedPackage(
             curation.base.apply(current)
         }.pkg.copy(
             // The declared license mapping cannot be reversed as it is additive.
-            declaredLicensesProcessed = DeclaredLicenseProcessor.process(pkg.declaredLicenses)
+            declaredLicensesProcessed = DeclaredLicenseProcessor.process(pkg.declaredLicenses),
+            // It is not possible to derive the original concluded license value with the above reversed application
```

Comment:
Nit: Blank line before this one to group the comment with the belonging line of code.